src/System.Diagnostics.Process/tests/ProcessCollectionTests.cs

```diff
@@ -59,6 +59,7 @@ public void TestModuleCollectionBehavior()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "https://github.com/dotnet/corefx/issues/22174")]
```

Comment:
If we plan to add uap versions of the tests later, personally I would put that in the comment instead of linking to the issue. It's just that usually skipping a test with a link of an issue means that you should use ActiveIssue and re-enable this later, but from what I see we don't plan on doing that.